lib/talkable/generators/install_generator.rb

```diff
@@ -37,20 +37,45 @@ def load_talkable_offer
 
       empty_directory "app/views/shared/"
 
-      if options[:haml] || options[:slim]
-        ext = options[:haml] ? 'haml' : 'slim'
+      if erb?
+        copy_file "app/views/shared/_talkable_offer.html.erb", "app/views/shared/_talkable_offer.html.erb"
+        inject_into_file "app/views/layouts/application.html.erb", before: "</body>" do
+          "<%= render 'shared/talkable_offer', offer: @offer %>\n"
+        end
+      else
+        ext = template_lang
 
         copy_file "app/views/shared/_talkable_offer.html.#{ext}", "app/views/shared/_talkable_offer.html.#{ext}"
         gsub_file "app/views/layouts/application.html.#{ext}", /^(\s*)\=\s*yield\s*$/ do |line|
           paddings = line.match(/(\s*)\=/)[1]
-          "#{line}#{paddings}= render 'shared/talkable_offer'\n"
+          "#{line}#{paddings}= render 'shared/talkable_offer', offer: @offer\n"
         end
+      end
+    end
+
+    def add_invite_controller
+      ext = template_lang
+
+      copy_file "app/controllers/invite_controller.rb", "app/controllers/invite_controller.rb"
+      copy_file "app/views/invite/show_offer.html.#{ext}", "app/views/invite/show_offer.html.#{ext}"
+      route "get '/invite' => 'invite#show_offer'"
+    end
+
+    protected
+
+    def template_lang
+      @template_lang ||= if options[:haml]
+        'haml'
+      elsif options[:slim]
+        'slim'
```

Comment:
I would not support slim - it is too rare. In case it makes too much time, lets drop it.